lib/membrane_rtc_engine/track/bitrate_estimation.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Membrane.RTC.Engine.Track.BitrateEstimation do
+  @moduledoc """
+  Struct describing bandwidth estimation.
+  """
+
+  @enforce_keys [:estimation, :error]
+  defstruct [:estimation, :error]
+
+  @typedoc """
+  Type describing bitrate estimation:
+  - estimation - estimated bitrate (calculated as bitrate taken up by average buffer)
+  - error - expected error of the estimation (calculated as standard deviation)
```

Comment:
```suggestion
  Type describing bitrate estimation:
  - estimation - estimated bitrate in kbps
  - error - expected error of the estimation 
```

I am not sure we should put how bitrate and error values are calculated here. The engine doesn't provide them and this struct could potentially be used by different endpoints in different ways.  So either delete how estimation and error are calculated or rename this module to `Membrane.RTC.Engine.Endpoint.WebRTC.BitrateEstimation`.

Also, why we define  `error` as `expected` error and not just as `error of the estimation calculation`